.github/workflows/test-linux-cpu.yml

```diff
@@ -0,0 +1,127 @@
+name: Unit-tests on Linux CPU
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+env:
+  CHANNEL: "nightly"
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python_version: ["3.7"] # , "3.8", "3.9", "3.10"
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.12xlarge
+      repository: pytorch/rl
+      script: |
+        pwd
+        ls -la
+
+        ## setup_env.sh
+        # Avoid error: "fatal: unsafe repository"
+        git config --global --add safe.directory '*'
+
+        script_dir=".circleci/unittest/linux/scripts"
+        root_dir="$GITHUB_WORKSPACE"
+        work_dir="/work"
+        conda_dir="${work_dir}/conda"
+        env_dir="${work_dir}/env"
+        lib_dir="${env_dir}/lib"
+
+        # 1. Install conda at ./conda
+        if [ ! -d "${conda_dir}" ]; then
+            printf "* Installing conda\n"
+            wget -O miniconda.sh "http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh"
+            bash ./miniconda.sh -b -f -p "${conda_dir}"
+        fi
+        eval "$(${conda_dir}/bin/conda shell.bash hook)"
```

Comment:
Those containers should already have conda installed

Reply:
@malfet Thanks for the help today. The workflow is green now. Moving forward as planned.